apps/admin-gui/src/app/shared/components/dialogs/add-edit-notification-dialog/add-edit-notification-dialog.component.ts

```diff
@@ -110,6 +138,18 @@ export class AddEditNotificationDialogComponent implements OnInit {
 
   save(): void {
```

Comment:
changes to plain text notifications do not take effect. Probably an issue with the form controls?

Reply:
Shouldn't the plain text input be considered as text input and therefore we don't need to validate it?

Reply:
Oh yeah I see the problem. My bad